keyboards/lets_split/keymaps/halvves/keymap.c

```diff
@@ -0,0 +1,265 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// smooth mousekeys just put this in your config (copied from the ergo ez config)
+#define MOUSEKEY_INTERVAL 16
+#define MOUSEKEY_DELAY 0
+#define MOUSEKEY_TIME_TO_MAX 60
+#define MOUSEKEY_MAX_SPEED 7
+#define MOUSEKEY_WHEEL_DELAY 0
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _COLEMAK 1
+#define _DVORAK 2
+#define _LOWER 3
+#define _RAISE 4
+#define _UTIL 5
+#define _MOUSE 6
+#define _ADJUST 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  COLEMAK,
+  DVORAK,
+  LOWER,
+  RAISE,
+  UTIL,
+  MOUSE,
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
```

Comment:
QMK defines these already, so they can be safely removed.

Reply:
Sorry about all of this. I made this keymap a while back based on (I think) the default keymap and haven't updated in a minute. 😔